diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/utils/AstNodeUtils.kt

```diff
@@ -1150,6 +1173,40 @@ fun doesLambdaContainIt(lambdaNode: ASTNode): Boolean {
     return hasNoParameters(lambdaNode) && hasIt
 }
 
+/**
+ * Checks that node (property or backing field) has pair node (backing field or property)
```

Comment:
what does it mean `has pair node`?

Reply:
It means that property node has matching backing field node or backing field node has matching property node. Nodes are matching if  they have matching names (xyz -> _xyz), have same types (but backing field can be nullable), were backing field is private, backing field should have no accessors/modifiers, property should have at least an accessor, or a modifier, or both. Added this in the description.